docs/communication_protocols/dronecan_protocol.rst

```diff
@@ -806,4 +806,161 @@ The following is performed on ArduCopter version v4.5.7
+
+Each `StatusExtended <https://github.com/dronecan/DSDL/blob/master/uavcan/equipment/esc/1036.StatusExtended.uavcan>`_ message sends exactly 56 bits of data, and only requires one full frame. This gives us about 131 bits of data.
+
+Each `EscStatus <https://github.com/dronecan/DSDL/blob/master/uavcan/equipment/esc/1034.Status.uavcan>`_ message transmits 110 bits of data, and requires 3 frames to transmit. 
+The first two frames send a full 7 bytes of data (plus the tail byte, and including the 2 CRC bytes in the first frame as discussed above), and the final frame sends only 3 
+bytes (including the tail byte). This means the first two frames are about 131 bits, and the final frame will have 91 bits.
+
+An example of a Vertiq module transmitting telemetry as viewed through the DroneCAN GUI tool is given below.
+
+.. image:: ../_static/comms_protocols_pictures/dronecan/telem_image.png
+
+In total, Vertiq module's DroneCAN telemetry requires:
+
+*131 bits StatusExtended + 353 bits EscStatus = 484 bits*
+
+So, we can find our utilization by:
+
+*484 bit/telem * 1/500Kbps = 968us* to transmit telemetry at 500kbit **per module**. Assuming our time period of 1s, we find:
+
+.. csv-table:: DroneCAN Telemetry Bus Utilization
+	:file: ../_static/comms_protocols_pictures/dronecan/telem_breakdown.csv
+	:header-rows: 1
+
+In general, the percentage of total bus utilization over one second for telemetry only can be best estimated by:
+
+*Utilization (%) = (484 bit/telemetry) * (Transmission Frequency (Hz)) * 1/Bitrate * (Number of Module) * 100%*
+
+.. _raw_cmd_utilization:
+
+Utilization With DroneCAN RawCommands
+===========================================
+
+The DroneCAN `RawCommand message <https://github.com/dronecan/DSDL/blob/master/uavcan/equipment/esc/1030.RawCommand.uavcan>`_ transmits one 14 bit command per each independent 
+ESC commanded. So, the number of frames necessary to transmit a full RawCommand varies on the number of commands being transmitted per message. Some practical examples, and 
+general formulae are provided below.
+
+Let's suppose that we need to control a generic quadcopter requiring only 4 independent commands. In this case, we only require 1 frame to send all of the required controls, 
+and will fill the data portion of the frame entirely. We can visualize this in the DroneCAN GUI tool as illustrated below:
+
+.. image:: ../_static/comms_protocols_pictures/dronecan/one_raw_cmd.png
+
+The result is a 131 bit frame. The utilization of a one second period is summarized in the table below:
+
+.. csv-table:: One RawCommand Bus Utilization
+	:file: ../_static/comms_protocols_pictures/dronecan/one_raw_cmd_util.csv
+	:header-rows: 1
+
+When using 4 or fewer ESC commands, we require only one frame to send all commands, and the utilization can be calculated by:
+
+*RawCommand Utilization (%) = (131 bit/command - ((4 - Number of Commands) * 8)) * (Command Frequency (Hz)) * 1/Bitrate * 100%*
```

Comment:
One random thing I only noticed now and I'm not sure if I quite understand, why is (4 - number of commands) * 8? The 8 part specifically is what I was unsure about. If its 14 bits per command, why shouldn't it be times 14 if this is all in terms of bits?